python/openravepy.__init__.py

```diff
@@ -142,7 +142,10 @@ def __unicode__(self):
         return s
         
     def __str__(self):
-        return unicode(self).encode('utf-8')
+        if sys.version_info[0]>=3:
+            return self.__unicode__()
+        else:
+            return unicode(self).encode('utf-8')
```

Comment:
this is kind of simulation of https://github.com/benjaminp/six/blob/1.16.0/six.py#L958 , we could use six instead

Reply:
sure

Reply:
@cielavenir feel free to make the change

Reply:
right I should have used six here, six is already used in some places..